ipc/src/stream_codec.rs

```diff
@@ -0,0 +1,137 @@
+extern crate tokio_core;
+
+use std::io;
+use stream_codec::tokio_core::io::{Codec, EasyBuf};
+
+pub struct StreamCodec;
+
+fn is_whitespace(byte: u8) -> bool {
+	match byte {
+		0x0D | 0x0A | 0x20 | 0x09 => true,
```

Comment:
Spaces (`0x20`) and Tabs (`0x09`) can occur in JSON strings, so doesn't it cause some issues with the condition in L48?